### DISCUSSION TITLE
Solidity language server support

### DISCUSSION THREAD

**Discussion by appetrosyan** [joaotavora/eglot#1304] [Ideas]
`aur/solidity-language-server` is a language server that I'd like to be able to use with `eglot`. There's a similar discussion for support on the `lsp-mode` repo, but that support seems to be incomplete and not ready for prime time.

**Comment by joaotavora** [comment]
If it's an LSP server, you probably already use it with Eglot, no need for
a pull request or a specialized package like in and other clients. Often
just finding the file, `M-x eglot` and pointing it to the program works.
